libsrc/_DEVELOPMENT/math/float/mathd32/c/sccz80/cd32_sccz80_dsneg.asm

```diff
@@ -0,0 +1,29 @@
+
+; float __fsneg (float number)
+
+SECTION code_clib
+SECTION code_math
+
+PUBLIC cd32_sccz80_fsneg
+
+EXTERN cd32_sccz80_dread, md32_fsneg, cd32_sccz80_dret
+
+.cd32_sccz80_fsneg
```

Comment:
I'd be very tempted to just xor the appropriate bit in the raw registers here. 

I've inlined this operation in sccz80 and I think sdcc does it too.

Reply:
Yes. It would be faster and neater.

But, until I get everything working, I'd prefer to keep linkages and work separate. That way I only have to fix broken things once.

Reply:
Actually, just to comment further. It is not a stretch to simply change the entry and return values for the intrinsic functions to match `DEHL`. And, I think that is the right thing to do. It will save needing to shuffle the stack because there will be no need to "post process" the library results with its complicating return address. The linkage can be done with jumps rather than calls.

But, let me get the rest of the library working, and come back for this change on the todo list.